pkg/api/v1alpha1/tinkermachineconfig.go

```diff
@@ -27,6 +135,7 @@ func NewTinkerbellMachineConfigGenerate(name string) *TinkerbellMachineConfigGen
 				Name:              "ec2-user",
 				SshAuthorizedKeys: []string{"ssh-rsa AAAA..."},
 			}},
+			TemplateOverride: fmt.Sprintf(TinkerbellDefaultTemplate, name, name),
```

Comment:
Add a TODO here to not include this in clusterconfig by default but use it if a user doesn't specify it